### PR TITLE
[FIX]#250 Failing config for fresh system

### PR DESCRIPTION
**Pull request by Imipenem**

### CODE REVIEW

File: mlf_core/config/config.py [hunk 1]
```diff
@@ -44,8 +45,9 @@ def config_general_settings() -> None:
         already_configured = False
         settings = {}
         if ConfigCommand.check_mlf_core_config_dir_exists() and 'GITHUB_ACTIONS' not in os.environ:
-            already_configured = True
-            settings = load_yaml_file(ConfigCommand.CONF_FILE_PATH)
+            if os.path.exists(Path(ConfigCommand.CONF_FILE_PATH)):
```
Comment (Zethson) [inline mlf_core/config/config.py:48]: Could add this to the check before?
a and b and c

Comment (Imipenem) [reply mlf_core/config/config.py:48]: Yeah thought about it and decided to put it here to reduce the length of the if statement and make it a bit more readable?

Comment (Zethson) [reply mlf_core/config/config.py:48]: I'd suggest to use line breaks and put it into one big IF

**Review by Zethson — APPROVED** [review]
LGTM

**Pull request merged**